tge/generate-merkle-tree.js

```diff
@@ -0,0 +1,51 @@
+const fs = require("fs");
+const ethers = require("ethers");
+const web3 = require("web3");
+const { default: MerkleTree } = require("merkletreejs");
+const keccak256 = require("keccak256");
+const airdropJson1 = require('./tge-airdrop-1.json');
+
+let airdropTree = {};
+let hashes = [];
+let totalAmount = web3.utils.toBN(0);
+
+const airdropJsons = [airdropJson1];
+
+for (const airdropData of airdropJsons) {
+    for (const beneficiary in airdropData) {
+        let amounts = airdropData[beneficiary]["token_eligibility"];
+        if (!airdropTree[beneficiary]) {
+            airdropTree[beneficiary] = "0";
+        }
+        airdropTree[beneficiary] = 
+            web3.utils.toBN(airdropTree[beneficiary]).add(
```

Comment:
I wouldn't be that specific as the names of those fields will change on each airdrop. I would iterate on each one of the fields